docs/install/oauth.md

```diff
@@ -0,0 +1,38 @@
+# GitHub OAuth
+
+By default, Coder is accessible via built-in authentication. Alternatively, you
+can configure Coder to enable logging in through GitHub OAuth. See below for
```

Comment:
Does this work with GitHub Enterprise or just GitHub? Might be nice to specify ;) 

cc @kylecarbs